CHANGELOG.md

```diff
@@ -42,6 +42,7 @@ _Changed ownership of the ta4j repository_: from mdeverdelhan/ta4j (stopped the
 - **AroonOscillatorIndicator**: new indicator based on AroonUp/DownIndicator
 - **AroonUpIndicator** and **AroonDownIndicator**: New constructor with parameter for custom indicator for min price and max price calculation
 - **ROCVIndicator**: rate of Change of Volume
+- **ZScoreIndicator**: the statistic z-score
```

Comment:
Please add your entry to the unreleased section. This was for the last release 0.10